projects_test.go

```diff
@@ -116,3 +116,50 @@ func TestAccCreateNonDefaultOrgProject(t *testing.T) {
 		t.Fatalf("Expected new project to not be part of org %s", orgPath)
 	}
 }
+
+func TestAccListProjects(t *testing.T) {
+	skipUnlessAcceptanceTestsAllowed(t)
+	c := setup(t)
+
+	defer projectTeardown(c)
+
+	rs := testProjectPrefix + randString8()
+
+	u, _, err := c.Users.Current()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	orgPath := "/organizations/" + u.DefaultOrganizationID
+
+	pcr := ProjectCreateRequest{Name: rs}
+	p, _, err := c.Projects.Create(&pcr)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if p.Organization.URL != orgPath {
+		t.Fatalf("Expected new project to be part of org %s, not %v", orgPath, p.Organization)
+	}
+
+	listOpt := &ListOptions{
+		Includes: "members",
+	}
+	projs, _, err := c.Projects.List(listOpt)
+	if err != nil {
+		t.Fatal(err)
+	}
+
```

Comment:
Do you think you could also test that pagination for the projects work? Like this test in the volumes:
https://github.com/packethost/packngo/blob/master/volumes_test.go#L153

We don't do that for device listing because device creation takes a long time, but project creation is instant.

You could write new test `TestAccProjectLargeList` where you create 3 project, and then list them with 
```
listOpt := &ListOptions{
	Page:    1,
	PerPage: 2,
}
```
.. check that the `len` is 2, and then list with

```
listOpt := &ListOptions{
	Page:    2,
	PerPage: 2,
}
```
.. and check that the `len` is 1.

Reply:
I added this -> https://github.com/packethost/packngo/pull/87/files#diff-ebeb10b961b2490421175ec9e1972041R75 
The pagination will not matter here since the List call will make sure to pull all projects  just like it is done for devices.

Reply:
@jasmingacic I added a test for pagination.